### PR TITLE
SelectBannerScreen full implementation

### PR DESCRIPTION
**Pull request by EphemeralX**
* SelectBannerScreen implementation
* CreateRoomScreen navigates to SelectBannerScreen, which will actually create the room.
* Add bannerImage as an attribute to the Room model.

I don't like using AsyncStorage to pass the bannerImage parameter but it didn't work with a parent setState function.

### CODE REVIEW

File: civilapp/src/browse/SelectBannerScreen.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    }
+
+    return (
+      <SuperTextHeader
+        leftButtonPress={backButtonPress}
+        rightButtonPress={this.updateBanner}
+        middleText={'Choose Banner'}
+        leftText={'<<'}
+        rightText={'Next'}
+        disabled={disabled}
```
Comment (carteryu) [inline civilapp/src/browse/SelectBannerScreen.js]: should rename this prop at some point, disabled is a super ambiguous name.

File: civilapp/src/browse/CreateRoomScreen.js [hunk 2]
```diff
@@ -42,44 +42,53 @@ export default class CreateRoomScreen extends React.Component {
...
     let { roomTitle, userInfo } = this.state
+
     const channel = await sbCreateOpenChannel(roomTitle)
     await sbOpenChannelEnter(channel)
 
     if (!userInfo) {
       userInfo = await AsyncStorage.getItem('USER_INFO')
       userInfo = JSON.parse(userInfo)
     }
+    const bannerImage = await AsyncStorage.getItem('BANNER_IMAGE')
```
Comment (carteryu) [inline civilapp/src/browse/CreateRoomScreen.js]: why store in async storage?

File: civilapp/src/browse/SelectBannerScreen.js (hunk 1 above)
Comment (EphemeralX) [reply civilapp/src/browse/SelectBannerScreen.js]: can enhance later as leftButtonHidden and rightButtonDisabled or something.

File: civilapp/src/browse/CreateRoomScreen.js (hunk 2 above)
Comment (EphemeralX) [reply civilapp/src/browse/CreateRoomScreen.js]: nav params didn't work, got a better idea?

**Pull request merged**